framework/libra-framework/sources/ol_sources/tests/boundary.test.move

```diff
@@ -210,4 +212,54 @@ module ol_framework::test_boundary {
     // aborts
   }
 
+
+  #[test(root = @ol_framework, alice = @0x1000a, marlon = @0x12345)]
+  fun test_epoch_trigger_disabled(root: &signer) {
+    common_test_setup(root);
+    // testing mainnet, so change the chainid
+    testnet::unset(root);
+
```

Comment:
I would add one assertion here to show that the feature flag is not enabled.

Reply:
added. thank you